### PR TITLE
New Extensions

### PR DESCRIPTION
**Pull request by eirannejad**

### CODE REVIEW

File: extensions/extensions.json [hunk 1]
```diff
@@ -110,6 +105,51 @@
...
+            "builtin": "False",
+            "default_enabled": "True",
+            "type": "extension",
+            "rocket_mode_compatible": "True",
+            "name": "pySSG",
+            "description": "IronPython Scripts for Autodesk Revit.",
+            "author": "Kyle Bruxvoort",
+            "author_profile": "https://kbrux.com",
+            "url": "https://github.com/kbruxvoort/SSG.extension.git",
+            "website": "https://archilizer.com",
```
Comment (jmcouffin) [inline extensions/extensions.json:146]: I think this is the wrong website address @eirannejad

Comment (eirannejad) [reply extensions/extensions.json:146]: Uh oh!

**Comment by eirannejad** [comment]
:package: New public release are available for [4.8.10.22040](https://github.com/eirannejad/pyRevit/releases/tag/v4.8.10.22040)

File: extensions/extensions.json (hunk 1 above)
Comment (kbruxvoort) [reply extensions/extensions.json:146]: I just now realized pySSG got added to the extensions list. Amazing!

**Pull request merged**